pyhap/accessory.py

```diff
@@ -143,6 +147,10 @@ def __init__(self, display_name, aid=None, mac=None, pincode=None,
         self.config_version = 2
         self.reachable = True
         self.pincode = pincode
+        if setup_id is None:
```

Comment:
The if-else i one line: `self._setup_id = setup_id or self._generate_setup_id()`. However, if you think the if-else is more readable, I don't mind.

Reply:
Yup, didn't know you could do the `or` in there like that. Learned something new!

Left unprotected, as technically, the accessory driver will need it for setup hash.